.travis.yml

```diff
@@ -16,6 +13,15 @@ install:
   - yarn install
   # puppeteer's install script downloads Chrome
 script:
-  - yarn run lint
-  - yarn run coverage
-  - yarn run test-phantom
+  - 'if [ "$NODE7" = "true" ]; then yarn run lint; fi'
+  - 'if [ "$NODE7" = "true" ]; then yarn run coverage; fi'
+  - 'if [ "$NODE7" = "true" ]; then yarn run test-phantom; fi'
+  - 'if [ "$NODE6" = "true" ]; then yarn run node6; fi'
+  - 'if [ "$NODE6" = "true" ]; then yarn run test-node6; fi'
+  - 'if [ "$NODE6" = "true" ]; then yarn run node6-sanity; fi'
```

Comment:
Why not simply make a shell script to call. Which will do the ENV check and run the right commands within that for the right one? Instead of having multiple commands each doing their own check.